clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fmap.py

```diff
@@ -0,0 +1,335 @@
+"""Module for converting field maps of ADNI."""
+from os import Path, PathLike
+from typing import List, Optional
+import pandas as pd
+
+def convert_adni_fmap(
+    source_dir: PathLike,
+    csv_dir: PathLike,
+    destination_dir: PathLike,
+    conversion_dir: PathLike,
+    subjects: Optional[List[str]] = None,
+    mod_to_update: bool = False,
+):
+    """Convert fMR images of ADNI into BIDS format.
+
+    Parameters
+    ----------
+    source_dir : PathLike
+        Path to the ADNI directory.
+
+    csv_dir : PathLike
+        Path to the clinical data directory.
+
+    destination_dir : PathLike
+        Path to the destination BIDS directory.
+
+    conversion_dir : PathLike
+        Path to the TSV files including the paths to original images.
+
+    subjects : List of str, optional
+        List of subjects.
+
+    mod_to_update : bool
+        If True, pre-existing images in the BIDS directory
+        will be erased and extracted again.
+    """
+
+    from os import path
+
+    import pandas as pd
+
+    from clinica.iotools.converters.adni_to_bids.adni_utils import paths_to_bids
+    from clinica.utils.stream import cprint
+
```

Comment:
You can cast your path variables to Path objects here and then work with them in downstream functions:

```suggestion
csv_dir = Path(csv_dir)
source_dir = Path(source_dir)
conversion_dir = Path(conversion_dir)
```

Reply:
Thanks I will do this